app/interactors/album.js

```diff
@@ -22,4 +22,14 @@ exports.purchaseAlbum = (user, album) => {
 
   });
 
+};
+
+exports.checkPurchasedAlbum = (userId, albumId) => {
+
+  return new Promise((resolve, reject) => {
+
+    Album.findOne({ where: {id: albumId, userId: userId} }).then(album => resolve(album));
```

Comment:
Same as the previous PRs, the orm is already returning a promise, so you don't need to create a new one.